../../Task/Exceptions-Catch-an-exception-thrown-in-a-nested-call/PascalABC.NET